docs/Jenkins & Docker.txt

```diff
@@ -0,0 +1,24 @@
+Jenkins and Docker are installed on the remote server so there is no need to have them on your local machine.
+To be able to operate them you have to receive your accounts. Once they are created for you ->
+Login to -web UI     https://bonnee.eu/ci/ 
+         -CLI        ssh accountName@bonnee.eu   
+         
+--Jenkins--
+The current Bonnie-Jenkins build is using Jenkinsfile located in the parent folder (Hexagonal).
+The branch source is: "dockerize-components" from "https://github.com/tamas-csanyi-cgi/bonnie" repository.
+
+To see all the available plugins on Jenkins from the web UI go to Dashboard>Manage Jenkins>Manage Plugins>Installed Plugins.
+To create a new build, click on Dashboard>New Item, then choose a name and type of the project. Fill out the configuration and click on Apply + Save. 
+Then click on Build Now, to start the building process.
+
+--Docker--
+The pipeline from Jenkinsfile contains, among other things, instructions to create docker images based on Dockerfiles located in the parent folder.
+Also, there is a "docker-compose.yml" file to automatically run created containers and the message services, so there is no need to do it manually.
+
+Basic CLI commands to operate docker images/containers manually:
+docker images - show all images.
+docker ps - show running images.
+docker run Image - create a new container of an image, and execute the container.
+docker stop Image - stop a continer.
```

Comment:
docker stop Image - stop a continer. ->docker stop containerName | containerId - stop a continer. *
* You can see container name or id as the output of 'docker ps' command